packages/react-components/react-carousel-preview/library/src/components/CarouselCard/CarouselCard.types.ts

```diff
@@ -7,7 +7,29 @@ export type CarouselCardSlots = {
 /**
  * CarouselCard Props
  */
-export type CarouselCardProps = ComponentProps<CarouselCardSlots>;
+export type CarouselCardProps = ComponentProps<CarouselCardSlots> & {
+  /**
+   * Sets the focus behavior for the card.
+   *
+   * `off`
+   * The card will not focusable.
+   *
+   * `no-tab`
+   * This behaviour traps the focus inside of the Card when pressing the Enter key and will only release focus when
+   * pressing the Escape key.
+   *
+   * `tab-exit`
+   * This behaviour traps the focus inside of the Card when pressing the Enter key but will release focus when pressing
+   * the Tab key on the last inner element.
+   *
+   * `tab-only`
+   * This behaviour will cycle through all elements inside of the Card when pressing the Tab key and then release focus
+   * after the last inner element.
+   *
+   * @default 'off'
+   */
+  focusMode?: 'off' | 'no-tab' | 'tab-exit' | 'tab-only';
```

Comment:
- Let's update to mentioned `CarouselCard`
- Also, do all of these values have sense? Should we start only with `off` & `tab-only`, for example?

Reply:
I think we should be ok here to enable these options and let users decide, while giving guidance in our story that 'tab-exit' is the likely best for default scenarios